addons/mail/static/src/components/attachment/attachment.js

```diff
@@ -94,10 +94,20 @@ class Attachment extends Component {
         if (this.detailsMode === 'card') {
             size = '38x38';
         } else {
-            size = '160x160';
+            // the size of background-image is depends on the props.imageSize
+            // to sync with width and height of preview div `o_Attachment_image`.
```

Comment:
don't need to say it is a "div", just the class name is enough (and I would add `.` before the classname to make it really clear we are talking about a class name)